target/scripts/start-mailserver.sh

```diff
@@ -260,20 +262,20 @@ source /usr/local/bin/daemons-stack.sh
 # ? >> Executing all stacks
 # ------------------------------------------------------------
 
-_notify 'tasklog' "Welcome to docker-mailserver $(</VERSION)"
+echo -e "Welcome to docker-mailserver $(</VERSION)\n"
 
 register_functions
 check
 setup
-[[ ${DMS_DEBUG} -eq 1 ]] && print-environment
+[[ ${LOG_LEVEL} =~ (debug|trace) ]] && print-environment
 fix
 start_misc
 start_daemons
 
 # marker to check, if container was restarted
 date >/CONTAINER_START
 
-_notify 'tasklog' "${HOSTNAME} is up and running"
+echo -e "\n${HOSTNAME} is up and running\n"
```

Comment:
```suggestion
_log 'info' "${HOSTNAME} is up and running"
```

Reply:
Another carry-over missing the commit message context ([original commit with context](https://github.com/docker-mailserver/docker-mailserver/pull/2486/commits/16b740e84e7433557cb6674ad601331adea21263)).

During original review, @georglauterbach added an `always` log level for log content that should persist through all levels, even when it wasn't fitting for that level. Such as an `info` that should log when using `error` or `warn` levels.

He later decided to drop the `always` log level, and just use a plain echo for such cases instead. I guess that has a little risk of other maintainers not knowing that and accidentally correcting to `info`, unless you think this should not be logged in `error` and `warn`?

Reply:
@polarathene is right, I want these messages to always be logged. I will add a comment to signal that these two are the only exceptions where `echo` is used instead of `_log`.

Reply:
Done in 2b9ab7c52949678ccdfc773a65744015c8cc989d

Reply:
Alright, but then it doesn't align and don't look uniformly/pretty:

```
[  TRACE  ]  Some very verbose trace message
[  DEBUG  ]  Some verbose debug message
[   INF   ]  Some info message

${HOSTNAME} is up and running

[ WARNING ]  Some warning message
[  ERROR  ]  Some error message
```

Then manually add INFO? `echo "[   INF   ]  ${HOSTNAME} is up and running"`

```
[  TRACE  ]  Some very verbose trace message
[  DEBUG  ]  Some verbose debug message
[   INF   ]  Some info message
[   INF   ]  ${HOSTNAME} is up and running
[ WARNING ]  Some warning message
[  ERROR  ]  Some error message
```

The same goes for the "welcome to DMS" message. We must decide, if we want a continuous look & feel, or not. I prefer a clean look.

Reply:
I would normally go for the second variant, but then it may be odd, especially when you set the log level to `warn` or `error`, that in info message is logged.

Therefore, I made a decision that should be comforting for all of us: I revised my earlier idea, and we will just log the message normally with `_log` on the `info` message. After all, these messages carry informational output, and they're not particularly special (after thinking about it twice; my idea was to always log them because they are in `start-mailserver.sh`). I hope this is now okay for everyone :)

Reply:
Done in [`1cbde1d`](https://github.com/docker-mailserver/docker-mailserver/pull/2493/commits/1cbde1d139f519716d0c439eb871fe7c37ded85d) and [`c098617`](https://github.com/docker-mailserver/docker-mailserver/pull/2493/commits/c098617f83d1fa2e1c3e0c9049267026b4b2e176)